src/components/Dropdown.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './dropdown.css';
+
+class Dropdown extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {display: 'Home'};
+        this.handleOption = this.handleOption.bind(this);
+    };
+
+    handleOption = (e) => {
+        e.preventDefault();
+        // this.state.display = e.target.display;
+        this.setState(this.state.display = e.target.display);
```

Comment:
ceva nu pare sa fie okey aici, daca cumva nu ii ceva new thing in React.

ex. Ai un state care arata asa: `{firstName: 'A', lastName: 'B' }` si vrei sa modifici firstName, atunci ai 2 posibilitati:
1. `this.setState({ firstName: 'some new value' })`
2. `this.setState(prevState => ({ firstName: 'some new value' }))` <- versiunea asta atunci il folosesti daca ai nevoie de ceva valoare din previous state (ex. cand vrei sa faci un increment la un counter care ii in state)

La setState ii destul sa ii dai in object doar proprietatiile care vrei sa fie modificate, si in background react lasa restul proprietati intacte.
*NOTE: la setState functionul din useState hook, ii putin diferit, acolo nu iti face merge la proprietatii, deci de exemplu daca ai in state un object, si face setState-ul asa cum ii in exemplul de sus, atunci lastName property nu va mai exista.